transmit-lib/src/cmr/transmit/metadata_db2.clj

```diff
@@ -49,6 +50,13 @@
   (when concept
     (update-in concept [:concept-type] keyword)))
 
+(defn include-request-id
```

Comment:
I would think that common already had something. Search has an "add-request-id-response-handler. there is a macro in common.api.context that may be relevant. Looks like the functions there are for creating a request id if one does not exist, but it may be the place to put things to address Tyler's suggestion.

Reply:
This function doesn't go into a response, it is really meant to put into the transmit calls to apps that get called from other apps, so that it is easier to see all of the logs from a front end user request.  I can put this function into cmr.transmit.http-helper.  That way all transmit classes have access to it.